pkg/pr/pr.go

```diff
@@ -33,7 +34,53 @@ func Execute(exe utils.Executor, options *Options) error {
 	return create(exe, options, branchID)
 }
 
+func filter(list []string, test func(string) bool) (ret []string) {
```

Comment:
This function is a very general utility function and should perhaps be moved somewhere else in the repo